greenmail-core/src/test/java/com/icegreen/greenmail/imap/commands/ImapProtocolTest.java

```diff
@@ -335,6 +335,52 @@ public void testRenameFolder() throws MessagingException {
         }
     }
 
+    @Test
+    public void testDeleteInbox() throws MessagingException {
+        store.connect("foo@localhost", "pwd");
+        try {
+            IMAPFolder folder = (IMAPFolder) store.getFolder("INBOX");
+            folder.open(Folder.READ_WRITE);
+
+            Response[] ret = (Response[]) folder.doCommand(protocol -> protocol.command("DELETE INBOX", null));
```

Comment:
Any reason why you don't use `folder.delete()` here?

Reply:
I assumed that in the file "ImapProtocolTest" we test the response to the IMAP command. If you think that it is ok just to call the delete method, how it should look (how can I check for exceptions being thrown)?

Reply:
Good point!